modules/ROOT/pages/mime-type-and-encoding.adoc

```diff
@@ -6,7 +6,7 @@ endif::[]
 
 include::partial$devkit-important.adoc[]
 
-In Mule 3.7 and newer, you can define the MIME type and encoding of a given return object from @Processor. See the http://mulesoft.github.io/mule-devkit/[Anypoint DevKit API Reference] for more information on annotations.
+In Mule 3.7 and newer, you can define the MIME type and encoding of a given return object from @Processor. 
```

Comment:
```suggestion
In Mule 3.7 and later, you can define the MIME type and encoding of a given return object from @Processor. 
```

Reply:
Font for @Processor correct?

Reply:
The rest of the doc has annotations with the regular font, so I'll leave it. This product will be archived soon